src/components/Landing/QAs.tsx

```diff
@@ -19,32 +34,74 @@ export default function QAs() {
     {
       q: t`What does Juicebox cost?`,
       a: [
-        t`Juicebox is an open protocol on Ethereum that is funded using Juicebox itself. You can check out the contractualized budget specs at https://juicebox.money/#/p/juicebox.`,
-        t`5% of all money made by projects is sent to help pay for the maintenance and development of Juicebox itself. In exchange, projects get Juicebox tokens ($JBX), which will be worth more as the ecosystem grows over time.`,
+        <Trans>
```

Comment:
**question**: why do we use `Trans` here but `t` on [line 62](https://github.com/jbx-protocol/juice-interface/pull/508/files#diff-5773f4d9ae3aea7f3f74b5b82ad78d77685f89277ccff3dbfbffa75342a263d6R62)

Reply:
Because the Trans tags have JSX elements inside them (namely `<OverflowVideoLink`), the `t`s are just strings

Reply:
awesome thank you!